### ISSUE TITLE
So how do I make "on_save" work exactly?

### ISSUE THREAD

**Issue by metaleap** [DisposaBoy/GoSublime#286]
ST2 on Win7x64.

So in the GoSublime.sublime-settings file we have:

```
// commands to run on (post) save - list of objects of the form {"cmd": "...", "args": {...}}
// Any TextCommand may be run. Supported GS commands include:
//     gs_comp_lint - compile the pkg and report any errors
"on_save": [],
```

Perfect! Just what I need. Problem is, no matter what kind of {"cmd":"foo"} I put inside the [square brackets], it's just ignored. Nothing is written to the ST2 console on file-save either, ever, other than the standard "Writing file /foo/bar.go" etc.

Now of course I have the default **ctrl+s** key-map of "gs_fmt_save", but does that really interfere with the _on_save_ handler? The two should surely co-exist peacefully?

Basically I want fmt-on-save and go-install-on-save: but **on_save=gs_comp_lint** didn't do anything (I verified with a new test pkg that is valid, compilable Go but does not get built into an **.a** object file inside $GOPATH/pkg/windows/etc.)

So I'm at a loss as to how to "shoehorn" GoSublime into the above behavior? The current "lint-as-you-type" is great, I certainly don't need or want "go-install-as-you-type", just "silently, on-save" would be great. ( _Silently_ because when I do _ctrl+b_, I run the "actual build process" -- any errors would show up, but in case of no errors, it would be blazing fast because all the various **.a** dependencies have already been compiled during my various _ctrl+s_ saves... )
